src/st/class-wpml-pb-integration.php

```diff
@@ -67,9 +67,9 @@ public function set_rescan( WPML_PB_Integration_Rescan $rescan ) {
 		$this->rescan = $rescan;
 	}
 
-	public function resave_post_translation_in_shutdown( WPML_Post_Element $post_element ) {
+	public function resave_post_translation_in_shutdown( WPML_Post_Element $post_element, $disallowed_in_shutdown = true ) {
```

Comment:
@dgwatkins, this function does not run in the shutdown. It enqueues a post that we want to resave in the shutdown.

Now, we need to enqueue posts while we are already in the shutdown, that's why I added this new argument.

FTR, this method was originally used by WPML Media via an action hook. If I remember, we were bailing out in shutdown to avoid infinite loop.

I would prefer not to change the naming now.

Reply:
The naming makes sense now
Thanks